plugin/audit_log/tests/mtr/audit_log_echo.inc

```diff
@@ -35,7 +35,7 @@ perl;
     }
     $line =~ s/"([a-zA-Z_ ]*)","([0-9]+)_[0-9_ :T-]*","[0-9_ :A-Z-]*"/"$1","<ID>","<DATETIME>"/;
     $line =~ s/"(Connect|Quit|Change user)","<ID>","<DATETIME>","[0-9]+"/"$1","<ID>","<DATETIME>","<CONN_ID>"/;
-    $line =~ s/"([A-Za-z ]+)","<ID>","<DATETIME>","([a-z_]+)","[0-9]+"/"$1","<ID>","<DATETIME>","$2","<CONN_ID>"/;
+    $line =~ s/"([A-Za-z ]+)","<ID>","<DATETIME>","(Ping|init db|Query|[a-z_]+)","[0-9]+"/"$1","<ID>","<DATETIME>","$2","<CONN_ID>"/;
```

Comment:
is Ping and Query suppose to have the initial as upper case here?

Reply:
Ping and Query will not be required here , [a-z_]  will handle that , I will remove these two from here , will keep "init db" so that I don't have to keep random space